docs/guides/algorithmiq-tem.ipynb

```diff
@@ -165,70 +167,145 @@
     "    qc.u(-1.2, -1.2, 0.3, 2)\n",
     "\n",
     "# Define the observables\n",
-    "observable = SparsePauliOp(\"YXZ\", 1.0)\n",
+    "observable = SparsePauliOp(\"IYX\", 1.0)\n",
     "\n",
     "# Define the execution options\n",
     "service = QiskitRuntimeService()\n",
     "backend_name = service.least_busy(operational=True).name\n",
+    "\n",
     "instance = \"<IQP_HUB/IQP_GROUP/IQP_PROJECT>\"\n",
     "\n",
-    "pub = (qc, observable)\n",
+    "pub = (qc, [observable])\n",
     "options = {\n",
-    "    \"backend_name\": backend_name,\n",
-    "    \"num_max_shots\": 100,\n",
+    "    \"max_execution_time\": 60,\n",
     "}\n",
-    "job = tem.run(instance=instance, pub=pub, options=options)"
+    "job = tem.run(pubs=[pub], instance=instance, backend_name=backend_name, options=options)"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "03998691",
+   "id": "6e2a97e2",
    "metadata": {},
    "source": [
-    "The following code checks your Qiskit Function workload's status or return results:"
+    "Use the Qiskit Serverless APIs to check your Qiskit Function workload's status:"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "856fe992",
+   "id": "47b0db71",
    "metadata": {},
    "outputs": [],
    "source": [
-    "print(job.status())\n",
-    "result = job.result()"
+    "print(job.status())"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "e2817b13",
+   "id": "fbaeace8",
    "metadata": {},
    "source": [
-    "## Advanced options\n",
-    "\n",
-    "You can fine-tune your calculations and get more verbose information by passing additional options to TEM."
+    "You can return the results as:"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 7,
-   "id": "c2ce65e3",
+   "execution_count": null,
+   "id": "ad5b9a53",
    "metadata": {},
    "outputs": [],
    "source": [
-    "import logging\n",
+    "result = job.result()"
```

Comment:
One code example should show how to extract the expectation value, e.g. `result[0].data.evs`. It can be this one, or the one in output, or both.

Reply:
All right, I added the suggested snippet to the notebook cell